src/redturtle/prenotazioni/browser/week.py

```diff
@@ -272,8 +291,31 @@ def get_booked_prenotation_message(self, day, booking):
         )
         return message
 
+    def get_prenotation_type(self):
+        """Returns the prenotation type passed by url"""
+        return self.request.get(TIPOLOGIA_PRENOTAZIONE_NAME, "")
+
+    def set_cookies(self):
+        self.set_prenotation_type()
+
+    def set_prenotation_type(self):
+        prenotation_type = self.get_prenotation_type()
+        cookie_value = self.request.cookies.get(TIPOLOGIA_PRENOTAZIONE_NAME_COOKIE, "")
+
+        if prenotation_type and not cookie_value:
```

Comment:
il fatto di avere una tipologia di prenotazione deve vincere sul fatto di avere un cookie. io toglierei `and not cookie_value`

Reply:
Di questo avevamo parlato con baio, visto che il cookie viene tolto quando visiti la pagina di prenotazione ho lasciato cosi